activation-registry/pom.xml

```diff
@@ -2,7 +2,7 @@
 
 <!--
 
-    Copyright (c) 2021, 2022 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2021, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
there are some updates required in this file around lines 55-67

Reply:
Thanks. To avoid more things like that, after running the next it only finds in the readme, which is expected.
```
angus-activation$ grep -R 'com.sun.activation.registries' ./
./docs/src/main/resources/README.md:Changes module name from `com.sun.activation.registries`
./docs/src/main/resources/README.md:`com.sun.activation.registries` to `org.eclipse.angus.activation`.
```